src/components/generic/OptionsList/OptionsList.jsx

```diff
@@ -15,6 +15,7 @@ import {useRef, useAutoFocus} from '../../hooks';
  * @param {string} props.selected  Selected option label
  * @param {Function} props.onSelect  A function which will be triggerd on option selection
  * @param {number} props.tabIndex  Value of the parent's tabIndex
+ * @param {Function} props.onBlur  A function that will close the optionList
```

Comment:
- square brackets as this is optional
- "Called when this component loses focus"